packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -258,7 +258,17 @@ function applySavedSettings(id: string, model: QueryModel): QueryModel {
             }
         }
 
-        return model.mutate(mutations as Partial<QueryModel>);
+        const modelWithSavedSettings = model.mutate(mutations as Partial<QueryModel>);
+
+        if (model.useSavedSettings === SavedSettings.noFilters) {
+            // If we're retrieving saved settings, but ignoring filters, we need to resave the settings without the
```

Comment:
I wonder if it might be worth opening an issue with a repro so that we can link this with automated regression coverage.

Reply:
I'm not sure it is, because the only place this is reproducible is the SampleListingPage, and only with the changes from this branch (if we don't include this fix). We don't use the SavedSettings.noFilters anywhere else.